app/src/main/java/ch/sdp/vibester/activity/TypingGameActivity.kt

```diff
@@ -223,6 +223,8 @@ class TypingGameActivity : GameActivity() {
         findViewById<LinearLayout>(R.id.displayGuess).removeAllViews()
         findViewById<EditText>(R.id.yourGuessET).text.clear()
         findViewById<EditText>(R.id.yourGuessET).isEnabled = true
+        val score = gameManager.getCorrectSongs().size
+        findViewById<TextView>(R.id.playerScore).text = "@string/your_score_dot_dot $score"
```

Comment:
it shows @string/your_score_dot_dot instead of "your score" when I tried it

Reply:
Will fix